src/huggingface_hub/commands/user.py

```diff
@@ -255,9 +255,10 @@ def run(self):
                 exit()
         try:
             url = self._api.create_repo(
-                self.args.name,
+                repo_id=self.args.name
+                if self.args.organization is None
+                else f"{self.args.organization}/{self.args.name}",
```

Comment:
(nit) can the if/else statement be taken out of the method call? I would find it easier to parse